src/features/tutorials/components/GenericTutorial.tsx

```diff
@@ -0,0 +1,113 @@
+import { t } from '@lingui/macro'
+import { useNavigation } from '@react-navigation/native'
+import LottieView from 'lottie-react-native'
+import AnimatedLottieView from 'lottie-react-native'
+import React, { FunctionComponent, useEffect, useRef } from 'react'
+import { Alert } from 'react-native'
+import styled from 'styled-components/native'
+
+import { UseNavigationType } from 'features/navigation/RootNavigator'
+import { _ } from 'libs/i18n'
+import { AnimationObject } from 'ui/animations/type'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiaryGreyDark } from 'ui/components/buttons/ButtonTertiaryGreyDark'
+import { StepDots } from 'ui/components/StepDots'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+type Props = {
+  animation: AnimationObject
+  animationSize: number
+  endFrame?: number
```

Comment:
Si je ne me trompe pas, quand on aura le tutoriel suivant, l'animation va "restart" au moment de quitter la page. Donc je suggère un naming qui n'inclut pas `end` :
```suggestion
  pauseAnimationOnRenderAtFrame?: number
```

Reply:
Je suis ok pour le renommage par contre je ne suis pas sûr de comprendre à 100% ton point pour le restart